integration-test/src/test/java/com/datastrato/gravitino/integration/test/catalog/jdbc/TestMultipleJdbcLoad.java

```diff
@@ -122,7 +122,7 @@ public void testCreateMultipleJdbc() throws URISyntaxException {
         postgreSqlCatalog
             .asSchemas()
             .listSchemas(Namespace.of(metalakeName, postgreSqlCatalogName));
-    Assertions.assertNotEquals(0, nameIdentifiers.length);
+    Assertions.assertEquals(0, nameIdentifiers.length);
```

Comment:
why check pg listSchemas returns empty while keeps  MySQL listSchema result not empty
```java

    NameIdentifier[] nameIdentifiers =
        mysqlCatalog.asSchemas().listSchemas(Namespace.of(metalakeName, mysqlCatalogName));
    Assertions.assertNotEquals(0, nameIdentifiers.length);
    nameIdentifiers =
        postgreSqlCatalog
            .asSchemas()
            .listSchemas(Namespace.of(metalakeName, postgreSqlCatalogName));
    Assertions.assertEquals(0, nameIdentifiers.length);
```